docs/mintlify/docs/overview/configuration.mdx

```diff
@@ -56,6 +63,45 @@ api_key = 'my-label-studio-api-key'
 | REPLICATE_API_TOKEN | [replicate]<br/>api_token | (string) API token to use for Replicate services |
 | TOGETHER_API_KEY | [together]<br/>api_key | (string) API key to use for Together AI services |
 
+## Rate Limit Configuration
+
+Pixeltable supports two patterns for configuring API rate limits in `config.toml`. Refer to the docstring of the relevant udf for details on the rate limiting pattern used by that udf.
+
+### Single Rate Limit per Provider
+
+For providers with a single rate limit across all models, add a `rate_limit` key to the provider's config section:
+
+```toml
+[mistral]
+api_key = 'my-mistral-api-key'
+rate_limit = 600  # requests per minute
+
+[fireworks]
+api_key = 'my-fireworks-api-key'
+rate_limit = 300
+```
+
+### Per-Model Rate Limits
+
+For providers that support different rate limits for different models, add a `<provider>.rate_limits` section and list the rate limits for each model:
+
+```toml
+[openai]
+api_key = 'my-openai-api-key'
+
+[openai.rate_limits]
+gpt-4o = 500
```

Comment:
Do these need to be quoted per the TOML spec?

Reply:
No.